tests/test_pointset.py

```diff
@@ -109,6 +109,7 @@ def test_filters_return_pointset(sphere):
 
 
 @pytest.mark.parametrize("force_float,expected_data_type", [(False, np.int64), (True, np.float32)])
+@pytest.mark.filterwarnings("ignore:Points is not a float type")
```

Comment:
Do we test anywhere that this warning is spit out? If not, I would think it is better to use the `pytest.warns` context manager for this.  Then you could also add in two more lines to test the default behavior, which would address your comment further down.

Reply:
Good point, thanks, it seems we don't check this warning anywhere. I think it's fair to add this in this PR, probably in the test that tests `force_float`, as you mentioned.